dask/dataframe/io/tests/test_parquet.py

```diff
@@ -188,6 +188,60 @@ def test_columns_index(tmpdir, write_engine, read_engine, columns, index):
     assert_eq(df[[]], ddf2)
 
 
+def test_index_with_multi_index(tmpdir, engine):
+    fn = os.path.join(str(tmpdir), 'test.parquet')
+    index = pd.MultiIndex.from_arrays([np.arange(10), np.arange(10) + 1],
+                                      names=['x0', 'x1'])
+    df = pd.DataFrame(np.random.randn(10, 2), columns=['a', 'b'], index=index)
+    df2 = df.reset_index(drop=False)
+
+    if engine == 'fastparquet':
+        fastparquet.write(fn, df, write_index=True)
+
+        # fastparquet doesn't support multi-index
+        with pytest.raises(ValueError):
+            ddf = dd.read_parquet(fn, engine=engine)
+    else:
+        import pyarrow as pa
+        pq.write_table(pa.Table.from_pandas(df), fn)
+
+        ddf = dd.read_parquet(fn, engine=engine)
+        assert_eq(ddf, df)
+
+        d = dd.read_parquet(fn, columns=['x0', 'a'], engine=engine)
```

Comment:
If I specify 'x0' as a column, I'd be surprised if it turns up in the index instead.